mod/metrics/lib/card/fixed_metric_answer_query.rb

```diff
@@ -48,5 +53,45 @@ def industry_query value
     def missing_answers
       FixedMetricMissingAnswerQuery.new(@filter_args).run
     end
+
+    def outliers
+      @restrict_to_ids[:answer_id] += savanna_outliers.keys
+    end
+
+    private
+
+    def outliers?
+      @filter_args[:metric_value] == :outliers
+    end
+
+    def id_value_map
+      all_related_answers.each_with_object({}) do |answer, h|
+        next unless answer.numeric_value
+        h[answer.id] = answer.numeric_value
+      end
+    end
+
+    def all_related_answers
+      Answer.where(metric_id: @metric_id)
+    end
+
+    # alternative method to determine outliers; not finished
+    def turkey_outliers
+      res = []
+      all_related_answers.map do |answer|
+        next unless answer.numeric_value
+        res << [answer.numeric_value, answer.id]
+      end
+      res.sort!
+      return if res.size < 3
+      quarter = res.size/3
+      q1 = res[quarter]
```

Comment:
as stated in the comment above the method: it's an alternative unfinished way to calculate outliers that I started in Cambridge but I stopped there because the time and effort/urgency/benefit ratio became to bad